### PR TITLE
Jie/hotspots fluctuation ready to merge with main branch

### PR DESCRIPTION
**Pull request by zhujie12138**
Implement the Thickness fluctuation and Hotspots fluctuation

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ CXXFLAGS      = -std=c++17 -Wall -fPIC -O3
 LD            = g++
 LDFLAGS       = -O3
 
-LIBS          = -I/usr/local/include -L/usr/local/lib -lgsl -lgslcblas -lcuba -lm `lhapdf-config --cflags --ldflags`#-lcuba -lm
```
Comment (thegarzo) [inline Makefile]: This needs to be returned to the original makefile.

File: configs/ipsat.yaml [hunk 2]
```diff
@@ -14,34 +14,43 @@ General:
...
       PDFSet: CT18NNLO
       ForcePositive: 1
-    K-Factor: 1.
+    K-Factor: 2.25
+    Fluctuations:
+        Thickness_fluct: False
+        Fluct_mode: Log_Normal
+        Sigma: 0.637
+        Hotspots_fluct: False
+        Bq: 0.04
```
Comment (thegarzo) [inline configs/ipsat.yaml:31]: Check what happens when Bq and Nq are not given.

**Review by thegarzo — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (zhujie12138) [reply Makefile]: Done

File: configs/ipsat.yaml (hunk 2 above)
Comment (zhujie12138) [reply configs/ipsat.yaml:31]: I Added few lines in nucleus.cpp to terminate the program when Nq, Bq or sigma are not given or inappropriate.

Comment (thegarzo) [reply configs/ipsat.yaml:31]: Ok , perfect

**Review by thegarzo — APPROVED** [review]

**Pull request merged**